test/integration/client-side-encryption/client_side_encryption.prose.deadlock.ts

```diff
@@ -35,8 +35,8 @@ const kClientsCreated = Symbol('clientsCreated');
 class CapturingMongoClient extends MongoClient {
   [kEvents]: Array<CommandStartedEvent> = [];
   [kClientsCreated] = 0;
-  constructor(url, options: MongoClientOptions = {}) {
-    options = { ...options, monitorCommands: true };
+  constructor(url: string, options: MongoClientOptions = {}) {
+    options = { ...options, monitorCommands: true, [Symbol.for('@@mdb.skipPingOnConnect')]: true };
```

Comment:
Out of curiosity, why was this flag added?

Reply:
Good question - I did forget to mention that in the PR description.  I'm happy to provide details if you want but the tl;dr is that when the Node driver connects with authentication enabled, it runs an extra `{ ping: 1 }` command.  Other drivers do not do this.  Consequently, if tests rely on command monitoring to assert on the correct ordering of events (like many driver tests do), the Node driver has an extra `ping` command so the assertions fail but only when authentication is enabled.

If you peruse the repo, you'll notice a lot of our tests are skipped when auth is enabled.  More often than not, that's why.

`skipPingOnConnect` was added last year by Neal, and it disables this ping for the purposes of testing.

Reply:
This was just a drive by cleanup to enable the tests for regular CI runs, instead of only the auth disabled tests.

Reply:
Cool! Thanks for the info. If you could just mention that in the '*_What's changing_*' section of the PR description, then I'd say we're good to go.